queries/prolog/highlights.scm

```diff
@@ -0,0 +1,59 @@
+(comment) @comment
+
+(atom) @constant
+
+((atom) @boolean
+  (#eq? @boolean "true"))
+
+((atom) @boolean
+  (#eq? @boolean "false"))
+
+(functional_notation
+  function: (atom) @function.call)
+
+[
+  (integer)
+  (negative_number
+    (integer))
+] @number
+
+[
+  (float_number)
+  (negative_number
+    (float_number))
+] @number.float
+
+[
+  (operator_1200xfx)
+  (operator_1200fx)
+  (operator_1100xfy)
+  (operator_1050xfy)
+  (operator_1000xfy)
+  (operator_900fy)
+  (operator_700xfx)
+  (operator_500yfx)
+  (operator_400yfx)
+  (operator_200xfx)
+  (operator_200xfy)
+  (operator_200fy)
```

Comment:
I think `":-"` and `"|"` should be added. Also `(comma)` should be given `@punctuation.delimiter` somewhere after this query

Reply:
I would maybe give `|` the same highlight group as its braces. Currently, only binary :- is colored as an operator, so I can color unary :- as one, too.